roles/openshift_master/tasks/set_loopback_context.yml

```diff
@@ -16,19 +16,30 @@
   - loopback_context_string not in l_loopback_config.stdout
   register: set_loopback_cluster
 
+- command: >
+    {{ openshift.common.client_binary }} config set-credentials
+    --client-certificate=/etc/origin/master/openshift-master.crt
+    --client-key=/etc/origin/master/openshift-master.key
+    --embed-certs=true
+    {{ openshift.master.loopback_user }}
+    --config={{ openshift_master_loopback_config }}
+  when:
+  - set_loopback_cluster | changed
+  register: l_set_loopback_credentials
+
 - command: >
     {{ openshift.common.client_binary }} config set-context
     --cluster={{ openshift.master.loopback_cluster_name }}
     --namespace=default --user={{ openshift.master.loopback_user }}
     {{ openshift.master.loopback_context_name }}
     --config={{ openshift_master_loopback_config }}
   when:
-  - set_loopback_cluster | changed
+  - set_loopback_cluster is changed
```

Comment:
Not entirely sure if ansible 2.4 supports `is` (we're using pipes almost everywhere)